### DISCUSSION TITLE
Installation Command Error

### DISCUSSION THREAD

**Discussion by dev-salam** [thedevdojo/voyager#5835] [General]
![Screenshot_4](https://github.com/thedevdojo/voyager/assets/114971278/3f7ed7f2-2c2e-45f2-ac34-5dd7a8494df9)

I am trying to install the Voyager, but it's not working for me is there any possible option?

**Comment by Emerica** [comment]
You don't appear to have installed voyager, what was the output of your composer require?
Does your composer.json actually contain a voyager entry?

**Reply by dev-salam** [reply]
Nope, i did  now, it's working thanks
